tests/draft2019-08/oneOf.json

```diff
@@ -112,6 +112,7 @@
     {
         "description": "oneOf complex types",
         "schema": {
+            "type": "object",
```

Comment:
Is this the same sort of issue for hypothesis-jsonschema? Can it get the same sort of "fix" :)?

Reply:
The *primary* principle here is that I really do think the meaning of this test is clearer with a type keyword.

It does have both valid and invalid cases, but runs into some performance issues. No type key implies that any type is valid, but any non-object instance will validate against both of the oneOf schemas. I intend to handle this and will be hand-writing many more tests, but thought that relying on an upstream teat incidentally doing this was not ideal.

Reply:
@Julian - I've removed this schema change in favor of writing up Zac-HD/hypothesis-jsonschema#19, which won't be too hard and is worth doing anyway.